packages/angular/build/src/builders/application/index.ts

```diff
@@ -259,4 +260,7 @@ function generateFullPath(
   return fullFilePath;
 }
 
-export default createBuilder(buildApplication);
+const builder: Builder<ApplicationBuilderOptions & json.JsonObject> =
```

Comment:
```suggestion
const builder: Builder<ApplicationBuilderOptions & import('@angular-devkit/core').json.JsonObject> =
```
I'd like to minimize the core package usage as much as possible here.  I think this is as far as possible right now. Assuming it works with the other changes.

Reply:
@devversion maybe we should switch to `Type` instead of `Interface` which would avoid the `& json.JsonObject`.

Reply:
It seems to be used throughout the repository, within the same builders. Are we good keeping as is and just leaving this to a general cleanup? (seems more work, even with Alan's idea; which seems to work; I'm not sure of the consequences and there would still be quite some cleanup work then)

Reply:
I'm fine with some TODOs here to cleanup later in the migration process.

Reply:
Discussed offline. TODO not necessary, but we'll track this in a separate issue as cleanup.